dehydrated

```diff
@@ -17,7 +17,7 @@ umask 077 # paranoid umask, we're creating private keys
 exec 3>&-
 exec 4>&-
 
-VERSION="0.7.1"
+VERSION="0.7.2"
```

Comment:
version change is maintainer only, not from pull requests

Reply:
Reverted that line.